cmd/imagedeps/main_test.go

```diff
@@ -37,16 +37,6 @@ func TestFunctional(t *testing.T) {
 		fn          tagFinderFn
 		expectError bool
 	}{
-		{
```

Comment:
not sure why this code is removed.

Reply:
because I was trying to remove minio entirely
I'm going to back out of doing that